handlers.go

```diff
@@ -3526,3 +3528,64 @@ func (s *Server) handleGetPublicNodeInfo(c echo.Context) error {
 		PrimaryAddress: s.FilClient.ClientAddr,
 	})
 }
+
+type retrievalCandidate struct {
+	maddr   address.Address
```

Comment:
these need to be exported (capitalized) otherwise the json formatter wont print them.

Probably also want to include the deal ID